webapp/src/tutorial.tsx

```diff
@@ -402,6 +407,7 @@ export class TutorialCard extends data.Component<TutorialCardProps, TutorialCard
 
         pxt.tickEvent(`tutorial.previous`, { tutorial: options.tutorial, step: previousStep }, { interactiveConsent: true });
         this.props.parent.setTutorialStep(previousStep);
+        this.setState({showUnusedBlockMessage: false});
```

Comment:
```suggestion
        this.setState({ showUnusedBlockMessage: false });
```

nit spacing are key/value pairs here and below